### PR TITLE
Only display week number if the week is configured to begin on Monday, fix #1780

### LINKED ISSUE #1780: Week numbers are wrong when "Week start" is set to Sunday
ISO 8601 weeks always start on Monday, and that is the week number that is displayed by the calendar, regardless of the "Week start" setting. The week number that is displayed for each week is the week number of the Sunday of the week.

This means that, in week-starts-on-sunday mode, week 1 of 2020 is 5 to 11 of January. This may or may not be correct for some week numbering system, but it is highly unlikely that any system would put week 53 of 2020 at 3 to 9 of January 2021, as it is currently displayed.

AFAIK there is no good standard for counting weeks starting on Sunday, so if that setting is on, the week numbers should be omitted altogether.

### PR DESCRIPTION
**Pull request by joshuaobrien**
Provides a fix for issue #1780

### CODE REVIEW

File: src/calendar/CalendarWeekView.js [hunk 1]
```diff
@@ -19,19 +19,19 @@ import {
 } from "./CalendarUtils"
 import {CalendarDayEventsView, calendarDayTimes} from "./CalendarDayEventsView"
 import {neverNull} from "../api/common/utils/Utils"
-import {getFromMap} from "../api/common/utils/MapUtils"
```
Comment (joshuaobrien) [inline src/calendar/CalendarWeekView.js:22]: This change is unrelated, I just noticed that the import wasn't used

**Comment by vitoreiji** [comment]
Hi @joshuaobrien, thanks for this. The week numbers are also shown in the month view, it would be nice if you could also apply your fix there :-)

**Comment by joshuaobrien** [comment]
> Hi @joshuaobrien, thanks for this. The week numbers are also shown in the month view, it would be nice if you could also apply your fix there :-)

Ah, I didn't realise. I'll update the month view tomorrow. Will tag you once the PR is ready for review again

**Comment by joshuaobrien** [comment]
Should be ready now @vitoreiji

**Comment by vitoreiji** [comment]
Sorry, I think I made a mess on your repo. I tried using the GH web UI to rebase your changes on top of the `calendar-week-improvements` but it looks like I merged that commit into your `master` instead. Can you please rebase your changes onto that branch and force update your `master`? Sorry for the trouble!

**Comment by joshuaobrien** [comment]
Sorry @vitoreiji, had some personal stuff on and missed this comment. I'll rebase tomorrow. Also, no problem, these things happen!

**Comment by joshuaobrien** [comment]
Done, @vitoreiji

**Comment by charlag** [comment]
thanks @joshuaobrien !

**Pull request merged**